src/swarm.js

```diff
@@ -125,7 +127,7 @@ module.exports = (common) => {
           }
 
           function getRepoPath () {
-            return '/tmp/.ipfs-' + Math.random().toString().substring(2, 8) + Date.now()
+            return path.join(os.tmpdir(), '.ipfs-' + Math.random().toString().substring(2, 8) + Date.now())
```

Comment:
While you are at it, could you do the same thing I requested at https://github.com/ipfs/ipfs-performance-profiling/pull/12#discussion_r154862400